### DISCUSSION TITLE
Use the original image on retry

### DISCUSSION THREAD

**Discussion by sheremet-va** [FRSOURCE/cypress-plugin-visual-regression-diff#162] [Q&A] [answered]
I've noticed that when test is retried, I get a new image for each retry. Is it possible to use the same image for comparing? For example, this test:

```ts
let retries = 0
it('retry', () => {
  cy.get('article').matchImage({
    title: 'custom',
  }).then(() => {
    retries++
    if (retries < 3) {
      throw new Error('error')
    }
  })
})
```

creates a new screenshot for each retry, when I already have screenshot in folder.

```ts
// cypress.config.ts
export default {
  retries: {
    runMode: 5,
    openMode: 0,
  }
}
```

**Comment by FRSgit** [comment, marked as answer]
Hey @sheremet-va, I believe this issue should be fixed in new release: 3.0.2 - please give it a try!
